visualCaseGen/custom_widget_types/case_creator.py

```diff
@@ -523,7 +523,7 @@ def _set_ntasks_ocean_based_on_grid(self, do_exec, min_points_per_core = 32, max
             cores = cores - iteration_amount
             pts_per_core = num_points/cores
 
-        xmlchange("NTASKS_OCN",cores, do_exec, self._is_non_local(), self._out)
+        xmlchange("NTASKS",cores, do_exec, self._is_non_local(), self._out)
```

Comment:
I think this function should only return `cores`, and should not call xmlchange. It should be up to callee to use the returned `cores` in an `xmlchange` call for NTASKS_OCN or whichever NTASKS_??? is appropriate (again, for generality purposes.) With this suggestion, this function can become a staticmethod since we wouldn't need `self` anymore (aside from logging, which can also happen outside this function). This would make this function compact and easy to unit test.